keyboards/keychron/q2/rev_0111/rules.mk

```diff
@@ -20,7 +20,8 @@ ENCODER_ENABLE = yes        # Enable Encoder
 DIP_SWITCH_ENABLE = yes
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = CKLED2001
-EEPROM_DRIVER = i2c
+# EEPROM_DRIVER = i2c
```

Comment:
Why this change?  

This would remove any eeprom support from the board and make via support useless.